### DISCUSSION TITLE
remove joomla in field colon: before value

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#31778] [General] [answered]
Steps to reproduce the issue

Create new field. Use this field.

Expected result

field withoutcolon value

Actual result

field withoutcolon: value

why : is displayed?

System information (as much as possible)

Safari on MacOS
PHP 8.0

Can you please provide more infos / screenshots how to put the : off ?

**Comment by ghost** [comment]
duplicate #31777

>   If you have a question on how to do something with Joomla! the [forum](https://forum.joomla.org/) is the best place to ask. 

> if you have an idea on how we can do something different/better you welcome to share your ideas in the GitHub discussions.

sources: #31704

**Comment by ghost** [comment]
Can you please provide more infos / screenshots how to put the : off ?

**Comment by ghost** [comment]
@chnnst [Joomla! forum](https://forum.joomla.org/) is the best place to ask

**Comment by ghost** [comment]
Joomla forum is the worst place to ask 

Can you please provide more infos / screenshots how to put the : off ?

**Comment by ghost** [comment]
i´m leaving this conversation as you don´t wanna ask on forum but repeat a question i can´t answer. good luck.

**Comment by ghost** [comment]
ok, good luck. 

I have an idea on how we can do field colon: before value different/better and I expected to be welcomed to share my ideas in the GitHub discussions

To theus who want to make joomla better please provide more infos / screenshots how to put the : off ?

**Comment by Bakual** [comment]
Your answer was posted three times already.
You can of course also check the documentation.

Please stop repeating the same question just because you don't like the correct answer or you will be threated as a troll.

**Comment by ghost** [comment]
There are no answer or infos / screenshots how to put the : off 

if joomla developers do not know how to make it, but have time to post threated as a troll answers , no wander why people drop joomla and go to use wordpress

good luck.

**Comment by infograf768** [comment, marked as answer]
You have to override the layout `/components/com_fields/layouts/field/render.php`
```
<?php if ($showLabel == 1) : ?>
	<span class="field-label <?php echo $labelClass; ?>"><?php echo htmlentities($label, ENT_QUOTES | ENT_IGNORE, 'UTF-8'); ?>: </span>
<?php endif; ?>
```
Locking as this question should have been posted on the forum.
